src/App.js

```diff
@@ -15,10 +15,34 @@ const tabs = [
     title: 'Tab 3',
     content: 'Some text 3',
   },
+  {
+    title: 'JSX here',
+    content: (
+      <div>
+        <h1>Checking JSX children</h1>
+        <button type="button" className="btn btn-dark">I guess working</button>
+      </div>
+    ),
+  },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+const preparedTabs = tabs.map((tab, index) => ({
+  ...tab,
+  id: index,
+}));
+
+class App extends React.PureComponent {
+  state = {
+    tabs: preparedTabs,
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Tabs tabs={this.state.tabs} index={1} />
```

Comment:
add startIndex to state
```suggestion
        <Tabs tabs={this.state.tabs} index={this.state.startIndex} />
```